src/AssetsTransferApi.ts

```diff
@@ -524,7 +581,11 @@ export class AssetsTransferApi {
 		const submittableData: SubmittableMethodData = JSON.parse(
 			submittableString
 		) as unknown as SubmittableMethodData;
-		const addr = submittableData.method.args.dest.id;
+
+		const addr: string = submittableData.method.args.target
```

Comment:
Hmm what will happen here if the string is empty? Will it ever reach that?

Reply:
great catch! I'll update this to cover the case for local transactions as well as V2 and V3 MultiAssets